XGBoost/XGBoost (1).py

```diff
@@ -0,0 +1,96 @@
+# XGBoost 
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt

```

Comment:
you don't need to import anything other than numpy,pandas for this class type implementation..

Reply:
done